### ISSUE TITLE
Jump to read receipt behaves unexpectedly

### ISSUE THREAD

**Issue by nadonomy** [vector-im/element-web#10270]
1. It isn't expected that clicking [Jump to read receipt] on user profiles should dismiss the profile and reveal the member list.

2. If you're jumping to a message at the bottom of the timeline, the message is highlighted with no way to dismiss it.

3. If you're jumping to a message at the bottom of the timeline, and you're scrolled above it, nothing happens.

**Comment by turt2live** [comment]
1. is https://github.com/vector-im/riot-web/issues/5760
2. sounds like a general permalink bug
3. sounds more like https://github.com/vector-im/riot-web/issues/2903

**Comment by nadonomy** [comment]
Ah thanks! I was just brain dumping with this issue. From a triage perspective— do you think it's worth me closing this issue and creating a new one for 2?

**Comment by turt2live** [comment]
possibly, although I'm not entirely decided on opening smaller issues for this. If we plan on going through the feature then it might be worth opening issues for each part, however if we don't plan on getting to it for a while then a dust collection issue might be better (which would then be split out when we actually get to implementation/design).

For me it's a larger question about issue triage: personally I find it difficult to aggregate all the gripes people have with a feature when N time has passed and there's Y open issues. It's much easier for me to crawl through a single issue with a few hundred comments to cut it apart. Although I think others work best in the complete opposite way :)

**Comment by nadonomy** [comment]
> For me it's a larger question about issue triage: personally I find it difficult to aggregate all the gripes people have with a feature when N time has passed and there's Y open issues. It's much easier for me to crawl through a single issue with a few hundred comments to cut it apart. Although I think others work best in the complete opposite way :)

For what it's worth, I'm aligned with you preferring a larger parent issue describing multiple smaller problems— I find it easier to discover issues via search being able to search by feature/product area, and also find it more productive when it comes to actioning them, as it's easier to conceptualise fixes holistically.

I'll leave this open for now and let someone more opinionated on triaging close or transpose this issue as they see fit.

**Comment by jryans** [comment]
> For me it's a larger question about issue triage: personally I find it difficult to aggregate all the gripes people have with a feature when N time has passed and there's Y open issues. It's much easier for me to crawl through a single issue with a few hundred comments to cut it apart. Although I think others work best in the complete opposite way :)

In general, I am personally in the camp that prefers many small separate issues so that each item can be easily tracked, has a clear "done" / "not done" status, is easy to independently link to and discuss, etc. However, it can be challenging to do "many small issues" with GitHub's current feature set since there's no native concept of a "meta issue". You can use labels to collect them all, assuming to remember to apply the labels everywhere you need them.

At the moment, I think it's best for the people working on a project / feature / area of code to organise their work in whatever way works well for them. 😄 As GitHub's features evolve, maybe a better approach will emerge in the future.

**Comment by afranke** [comment]
This is a terrible place to have a discussion about triaging issues. For the record, I agree with @jryans: compound issues make it difficult for people looking at them in the future to know what’s left to fix and the current status of the issue without having to read the whole discussion. This makes the triage issue worse for everyone.

---

The only remaining “subissue” here seems to be fixed:

1. I clicked on such a read receipt that was at the bottom of the timeline. The message got highlighted, with the profile of the member still visible in the right panel.
2. I closed the panel, the highlight got removed.

Closing as nothing remains to be fixed.
